generator.yaml

```diff
@@ -642,14 +642,22 @@ resources:
         - InvalidAction
         - UnrecognizedClientException
     hooks:
+      sdk_read_one_pre_set_output:
+        template_path: notebook_instance/sdk_read_one_pre_set_output.go.tpl
+      sdk_read_one_post_set_output: 
+        template_path: notebook_instance/sdk_read_one_post_set_output.go.tpl
+      delta_pre_compare:
+        code: customSetDefaults(a, b)
+      sdk_update_pre_build_request:
+        template_path: notebook_instance/sdk_update_pre_build_request.go.tpl
+      sdk_update_post_build_request:
+        code: handleUpdateOnlyParameters(desired, latest, input)
+      sdk_update_post_set_output:
```

Comment:
this is only called after a successful update operation, correct ?

Reply:
Yes if you are referring to sdk_update_post_set_output.